robottelo/ui/subscription.py

```diff
@@ -30,9 +31,12 @@ def upload(self, path, repo_url=None):
             self.text_field_update(locators['subs.repo_url_update'], repo_url)
             self.click(common_locators['save'])
         browse_element = self.wait_until_element(locators['subs.file_path'])
-        browse_element.send_keys(path)
+        with open(manifest.filename, 'wb') as handler:
```

Comment:
Every open operation will happen on the machine where robottelo is running. In order to do a file operation to the Satellite server an ssh operation is needed.